File: Programmers/Lv0/합성수찾기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int cnt = 0;
    for (int i = 1; i <= n; i++)
    {
        int divCnt = 0;
        for (int j = 1; j <= i; j++)
            if (i % j == 0)
                divCnt++;
        if (divCnt >= 3)
            cnt++;
    }
    return cnt;
}

File: Programmers/Lv0/합성수찾기.py
def solution(n):
    cnt = 0
    for i in range(1, n + 1):
        divCnt = 0
        for j in range(1, i + 1):
            if i % j == 0:
                divCnt += 1
        if divCnt >= 3:
            cnt += 1
    return cnt